starburst.gemspec

```diff
@@ -15,15 +15,15 @@ Gem::Specification.new do |s|
   s.license     = "MIT"
 
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.rdoc"]
-  
-  s.add_dependency "rails", ">= 3.0"
 
-  s.add_development_dependency "sqlite3"
+  s.add_dependency "rails", ">= 4.2.0", "< 6.0"
+
+  s.add_development_dependency "sqlite3", "~> 1.3.9"
 
   #RSpec
   s.test_files = Dir["spec/**/*"]
-  s.add_development_dependency "rspec-rails"
-  s.add_development_dependency "capybara"
-  s.add_development_dependency "factory_girl_rails"
-  s.add_development_dependency "appraisal"
+  s.add_development_dependency "rspec-rails", "~> 3.8.0"
+  s.add_development_dependency "capybara", "~> 2.4.1"
+  s.add_development_dependency "factory_girl_rails", "~> 4.4.1"
+  s.add_development_dependency "appraisal", "1.0.2"
```

Comment:
I've switched to set the development version requirements here instead of relying on the `Gemfile.lock`.